api/types/src/convert.rs

```diff
@@ -212,24 +224,34 @@ impl<'a, R: MoveResolver + ?Sized> MoveConverter<'a, R> {
         access_path: AccessPath,
         op: WriteOp,
     ) -> Result<WriteSetChange> {
+        let sk = StateKey::AccessPath(access_path);
```

Comment:
nit: `sk` being ultra short is so outstanding compared to all other names around here -_-